transports/azure-service-bus/configuration/full.md

```diff
@@ -46,8 +51,7 @@ The topology will create entities it needs, based on the following settings:
 
 The following settings are available to define how queues should be created:
 
- * `MaxSizeInMegabytes(SizeInMegabytes)`: The size of the queue, in megabytes. Defaults to 1,024 MB.
- * `MaxDeliveryCount(int)`: Sets the maximum delivery count, defaults to the number of Immediate Retries + 1. In case Immediate Retries are disabled, the transport will default `MaxDeliveryCount` to 10 attempts.
+ * `MaxSizeInMegabytes(SizeInMegabytes)`: The size of the queue, in megabytes. Defaults to 1,024 MB. 
  * `LockDuration(TimeSpan)`: The period of time that Azure Service Bus will lock a message before trying to redeliver it, defaults to 30 seconds.
  * `ForwardDeadLetteredMessagesTo(string)`: Forward all dead lettered messages to the specified entity. This setting is off by default.
```

Comment:
I believe this should be:
**forwards** all dead lettered messages to the specified entity. This setting is **set to** off by default. 

NOTE: if you change this to "set to off" then it should be changed throughout the text